.github/workflows/test.yml

```diff
@@ -40,11 +40,11 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip setuptools wheel
-        python -m pip install 'tox<4.0.0' 'tox-gh-actions==2.12.0'
+        python -m pip install 'tox~=4.0' 'tox-gh-actions~=3.0'
```

Comment:
Can you actually switch us to the first-party [tox-dev/tox-gh](https://github.com/tox-dev/tox-gh) instead?

Just requires changing `[gh-actions]` to `[gh]` in the ini file.